### ISSUE TITLE
feat: add SIOP v2 and OIDC for Verifiable Presentations.

### ISSUE THREAD

**Issue by zbrumen** [JanssenProject/jans#1172]
**Is your feature request related to a problem? Please describe.**
Currently, the janssen project and from what I have gathered, any OpenID server implementation, has no integration of SIOP or Verifiable Presentations of any kind. 

**Describe the solution you'd like**
Add SIOP v2 and OIDC for Verifiable Presentations.

**Describe alternatives you've considered**
I have created a custom small serverlet for verifing OpenID SIOP, and all we have is Demo's. I think integration into a 
large and stable project like jans, can be beneficial for the whole OIDC SSI auth flow :smile:

**Additional context**

**Comment by ossdhaval** [comment]
Yes, so we will keep this issue as the starting point of this effort. 

@zbrumen Next step is to create another issue using `development item` template. This new issue will be where all the action will happen. This new issue is where
- You detail out proposed design approach. And relevant code owners will help you refine it via review comments.
- As design becomes more and more solid, you'll be able to identify development tasks. You can create these tasks within this new issue and create separate issues from those tasks if required.
